tests/metrics/estimate_gas.py

```diff
@@ -638,19 +668,23 @@ def estimate_gas(analyzer: AnalyzeGas = None) -> None:
     log.info("Divide stake (next period is confirmed) = " + str(
         miner_functions.divideStake(0, MIN_ALLOWED_LOCKED, 2).estimateGas({'from': ursula1})))
 
+    #
     # Slashing tests
+    #
     tx = miner_functions.confirmActivity().transact({'from': ursula1})
     testerchain.wait_for_receipt(tx)
     testerchain.time_travel(periods=1)
-    # Deploy adjudicator mock to estimate slashing method in MinersEscrow contract
+    # Deploy adjudicator to estimate slashing method in MinersEscrow contract
     adjudicator, _ = testerchain.interface.deploy_contract(
         'MiningAdjudicator', miner_agent.contract.address, ALGORITHM_SHA256, MIN_ALLOWED_LOCKED - 1, 0, 2, 2
```

Comment:
Phew - where can I find some docs on these ints?

Reply:
Right now only in `MiningAdjudicator` contract and its tests
I removed `mock` word because forgot to do it before